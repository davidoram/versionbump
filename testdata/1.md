# Change log

## 0.0.2

- Update Postgres to version 13.3 [link](https://ticket/123)
- Added CHANGELOG.md
- Updated Ruby version to `2.7.3`
- Fixed audit vulnerabilities via `bundle update rdoc rake rack json activesupport activerecord`
## 0.0.1

Initial version