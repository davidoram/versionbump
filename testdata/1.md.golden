# Change log

## 0.1.0

This text is added to the top of the file [ticket](http://link/to/a/ticket)

## 0.0.2

- Update Postgres to version 13.3 [link](https://ticket/123)
- Added CHANGELOG.md
- Updated Ruby version to `2.7.3`
- Fixed audit vulnerabilities via `bundle update rdoc rake rack json activesupport activerecord`
## 0.0.1

Initial version